src/WrapperApp/components/Results/EstimatorsSelect/EstimatorsSelect.tsx

```diff
@@ -0,0 +1,111 @@
+import { Card, CardContent, Tab, Tabs } from '@mui/material';
+import { SyntheticEvent, useCallback, useState } from 'react';
+
+import { FullSimulationData, useShSimulation } from '../../../../services/ShSimulatorService';
+
+interface EstimatorsTabProps {
+	tabsValue: number;
+	setTabsValue: (value: number) => void;
+	simulation: FullSimulationData;
+	setResultsSimulationData: React.Dispatch<React.SetStateAction<FullSimulationData | undefined>>;
+	estimatorsTabData: string[];
+}
+
+const EstimatorsSelect = ({
+	tabsValue,
+	setTabsValue,
+	simulation,
+	setResultsSimulationData,
+	estimatorsTabData
+}: EstimatorsTabProps) => {
+	const [controller] = useState(new AbortController());
+
+	const { getJobResult } = useShSimulation();
+
+	const handleEstimatorTabChange = useCallback(
+		async (id: number) => {
+			const currentEstimatorData = simulation.estimators;
+			const estimatorExists = currentEstimatorData.some(
+				estimator => estimator.name === estimatorsTabData[id]
+			);
+
+			if (!estimatorExists) {
+				const simulationJobId = simulation.jobId;
+
+				const estimatorData = await getJobResult(
+					{
+						jobId: simulationJobId,
+						estimatorName: estimatorsTabData[id] + '_'
```

Comment:
why do we need an underscore here ?

Reply:
@SzymanskiBartlomiej told me to use estimatorName with underscore

Reply:
it might some bug in the https://github.com/DataMedSci/pymchelper library which wrongly appends `_` to the estimator name.  I wanted to cross-check if that is of your design or something that you figured out was there.

If you see some surprising design please don't accept it blindly, but communicate it (i.e. on chat) - this way we can spot and trace bugs.

Reply:
@SzymanskiBartlomiej why do we have to recive estimatorName with "_" on backend?

Reply:
The reason we have to use underscores in estimatorName is because this is how estimators are stored in the database.

Reply:
Try to understand why they are scored like that :)